modules/ROOT/pages/tuning-backend-server.adoc

```diff
@@ -0,0 +1,19 @@
+= Backend Server Response Time
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+When investigating performance issues, you should first look if the overall flow's performance is heavily constrained by the latency (response time or message round-trip time in milliseconds unless otherwise noted) of the remote hosts to which the flow connects into.
```

Comment:
I suggest omitting all the rest of this in favor of the rewording suggested above.
But I have copyedited the existing text in case you are uncomfortable rewording right now (but please save in a JIRA for future consideration).